### PR TITLE
feat(content): Additional spaceport descriptions during Remembrance Day

### PR DESCRIPTION
**Pull request by Anarchist2**
## Summary
This is a re-PR of #6516, updated to use newer, less save file bloat-inducing mechanics. Affected planets include almost everything with the `dirt belt` attribute, Alexandria, Allhome, pre-war Poisonwood and Valhalla. Remembrance Day effects appear from a fortnight before the day to 4 days after it.

## Save File
This save file can be used to test these changes:
[Rembrandt Dae~Remembrance Day Start.txt](https://github.com/user-attachments/files/17350093/Rembrandt.Dae.Remembrance.Day.Start.txt)

### CODE REVIEW

File: data/map planets.txt [hunk 1]
```diff
@@ -487,6 +504,10 @@ planet Bounty
...
 	description `	The focus on skilled manufacturing has had a dramatic effect on both the local education system and the sports scene. One can garner great respect - and a sizable income - simply by being the best at folding tiny paper cranes by microscope.`
 	spaceport `The spaceport is not fancy, but it is serviceable: the landing pads are very level and show only minor signs of wear and tear, and the one warehouse is tiny but recently built, and its roof is in good repair. Next to it is a marketplace building, with three walls and a pitched roof, and the fourth side open for people and trucks to come and go. An enterprising cook is selling lamb kebabs from a small grill and cart in one corner of the building.`
+	spaceport `	A moderately-sized festival has been set up in commemoration of Remembrance Day. Both locals and foreigners have gathered to show off their precision origami skills, with displays ranging from miniature Firebirds to hundreds of thousands of paper cranes, each representing a life lost in the Alpha Wars.`
```
Comment (TheGiraffe3) [inline data/map planets.txt:507]:
```suggestion
	spaceport `	A moderately-sized festival has been set up in commemoration of Remembrance Day. Both locals and off-worlders have gathered to show off their precision origami skills, with displays ranging from miniature Firebirds to hundreds of thousands of paper cranes, each representing a life lost in the Alpha Wars.`
```

File: data/map planets.txt [hunk 2]
```diff
@@ -1788,6 +1817,10 @@ planet Gemstone
...
 	spaceport `The spaceport is perched on top of a peak in an older and more stable mountain range. The landing pads here are used both by starships and by the helicopters that bring cargo to and from the mines. Overhead, helicopters fly by carrying heavy loads dangling from cables. A few low concrete buildings serve as warehouses, and on the far end of the port, where an explosion would do the least damage if a sudden, severe earthquake struck, large steel tanks hold deuterium for refueling ships. Asphalt roads with flimsy guard rails connect the landing pads to the warehouses.`
+	spaceport `	In commemoration of Remembrance Day, a few flags have been set up in the spaceport. You suspect that the risk of earthquake puts the locals off of setting up anything more elaborate.`
```
Comment (TheGiraffe3) [inline data/map planets.txt:1820]:
```suggestion
	spaceport `	In commemoration of Remembrance Day, a few flags have been set up in the spaceport. You suspect that the risk of an earthquake puts the locals off of setting up anything more elaborate.`
```

File: data/map planets.txt [hunk 3]
```diff
@@ -2243,6 +2280,10 @@ planet Hopper
 	description `	The oceans are populated only by a few indigenous life forms that have adapted to their high acidity; Earth fish could not be introduced successfully here.`
 	spaceport `The spaceport consists mostly of a cluster of steel-framed warehouses with tin roofs, surrounded by concrete slabs for ships to land on. Scattered among the warehouses are a few smaller wooden buildings, housing a pub, a restaurant, and a small inn.`
 	spaceport `	Some of the warehouses are relatively new, with metal that still gleams; others are covered in rust, and a few stand in ruins, with large holes worn through their roofs and rusted girders poking through like dead trees.`
+	spaceport `	The inn has been repurposed for hosting Remembrance Day celebrations. A canvas just above its entrance reads, "For those who have fallen, and those who may fall." The canvas is already blotchy.`
```
Comment (TheGiraffe3) [inline data/map planets.txt:2283]:
```suggestion
	spaceport `	The inn has been repurposed for hosting Remembrance Day celebrations. A canvas just above its entrance reads, "For those who have fallen, and those who may fall." It is already blotchy.`
```
Or also blotchy with what?
```suggestion
	spaceport `	The inn has been repurposed for hosting Remembrance Day celebrations. A canvas just above its entrance reads, "For those who have fallen, and those who may fall." It is already blotchy with tears.`
```

File: data/map planets.txt [hunk 4]
```diff
@@ -3301,6 +3346,10 @@ planet "New Argentina"
...
 	description `	Cattle handlers here do their work on horseback, because they believe modern vehicles are more likely to spook the animals.`
 	spaceport `The spaceport is on the outskirts of one of New Argentina's largest towns... which has a population of barely ten thousand people. Many of the dock workers are teenagers or barely past their teens, children of ranchers who are not drawn to the solitary sort of life that their parents enjoy. Aside from the church and the hospital, the largest building in town is "The Grange," a function hall with a sign out front advertising square dances and line dances three nights a week.`
+	spaceport `	The Grange is currently set up for Remembrance Day celebrations. The population inside consists mostly of the older residents of the planet, many of them likely having traveled tens of thousands of kilometers to arrive here.`
```
Comment (TheGiraffe3) [inline data/map planets.txt:3349]: Does ES use kilometers or miles?

File: data/map planets.txt [hunk 5]
```diff
@@ -3316,6 +3365,10 @@ planet "New Austria"
 	description `	For centuries the locals have been trying to find a deposit of diamonds, which would sell for much higher prices than sapphires, and local folklore revolves around the prosperity that would come to this world if diamonds were ever discovered.`
 	spaceport `The spaceport is nothing but a couple of landing pads on the outskirts of one of the larger mining villages, high up on a mountain ridge to allow easy access for starships. A few roads lead down from there to the village, where you can trade for local goods or get food from the miners' cafeteria.`
 	spaceport `	By the time you have walked to the center of the town, several people have already tried to sell you large "diamonds" that you strongly suspect are just clear sapphires.`
+	spaceport `	The village itself is abuzz for Remembrance Day. Various exhibits have been set up, displaying either replicas of the planet's largest sapphires, or the real deals. You recall seeing a cheesy Remembrance Day movie about a heist for one such sapphire when you were younger.`
```
Comment (TheGiraffe3) [inline data/map planets.txt:3368]: So what about other starts?
Would this still be the case?

File: data/map planets.txt [hunk 6]
```diff
@@ -3386,8 +3447,12 @@ planet "New Greenland"
 	description `New Greenland was first settled about six hundred years ago, at the peak of the "Turf Wars" era where most planets were controlled by powerful corporations and their struggles often turned into outright warfare with privately owned fleets and armies. The people who came to New Greenland were looking for a world so marginal and undesirable that they could live here in peace without being caught in the crossfire.`
 	description `	The locals here grow food in greenhouses and specialize in making warm winter clothing, which is this planet's only export.`
 	spaceport `The spaceport is near the equator, but it is still cold enough that there is always some snow on the ground.`
-	spaceport `	You have never seen as many people in fur coats as there are here; what might be considered a luxury in other parts of the galaxy is a necessity here, where the cheapest clothing material is the pelts of indigenous animals.`
+	spaceport `	You have never seen as many people in fur coats as there are here; what might be considered a luxury elsewhere is a necessity here, where the cheapest clothing material is the pelts of indigenous animals.`
```
Comment (TheGiraffe3) [inline data/map planets.txt:3450]: ~~unrelated change~~
And also, I'm not sure this is better; elsewhere could make it sound as if elsewhere on-planet.
```suggestion
	spaceport `	You have never seen as many people in fur coats as there are here; what might be considered a luxury in other parts of the galaxy is a necessity here, where the cheapest clothing material is the pelts of indigenous animals.`
```

File: data/map planets.txt [hunk 7]
```diff
@@ -3844,6 +3949,10 @@ planet Poisonwood
 	description `	Today, most of the inhabitants are children of the liberated slaves, and like their parents they remain grateful and fiercely loyal to the Republic.`
 	spaceport `The starport is a hodgepodge of small metal structures, built from the remains of Navy and pirate ships that fought for control of this world. The locals are unusually courteous and proud to welcome you to their planet.`
 	spaceport `	The tavern in the center of the port is named The Parting Glass. The walls are covered with photographs from after the war: liberated slaves boarding Navy ships to leave this planet that had brought them so much pain; Navy officers leaving their ships to retire on this world that they liberated; friends gone away, and friends left behind. It is a strange feeling to sit here so deeply immersed in signs of a history that you had no part in.`
+	spaceport `	A sign adverties a Remembrance Day special for half-price drinks.`
```
Comment (TheGiraffe3) [inline data/map planets.txt:3952]:
```suggestion
	spaceport `	A sign advertises a Remembrance Day special for half-price drinks.`
```

**Review by TheGiraffe3 — COMMENTED** [review]
These are really nice!

File: data/map planets.txt (hunk 4 above)
Comment (reticent-rem) [reply data/map planets.txt:3349]: Generally kilometers from what I can tell -- I found dozens of references to kilometers in human news/missions/campaign, but only one use of miles for a line of news.

File: data/map planets.txt [hunk 8]
```diff
@@ -773,6 +794,14 @@ planet Clark
...
 	spaceport `The spaceport is at the heart of one of Clark's largest cities, surrounded by smokestacks and factories. There are a few hangars owned by different corporations where their cargo ships can load and unload, but you are relegated to a set of concrete landing pads farther from the main buildings. Large trucks rumble by carrying carts loaded down with cargo crates.`
 	spaceport `	In the central building, dozens of workers fill a large cafeteria with several stations serving different types of food. The mood here feels boisterous and energetic, very different from most of the Dirt Belt worlds you have visited.`
+		to display
+			or
+				"days since year start" <= 35
+				"days since year start" >= 55
+	spaceport `	In the central building, dozens of workers fill a large cafeteria with several stations serving different types of food. The mood here feels boisterous and energetic, bolstered further by Remembrance Day celebrations. Locals and tourists take turns downing as many beers as possible as their friends shout Alpha Wars shanties.`
```
Comment (reticent-rem) [inline data/map planets.txt:801]:
```suggestion
	spaceport `	In the central building, dozens of workers fill a large cafeteria with several stations serving different types of food. The mood here feels boisterous and energetic, bolstered further by Remembrance Day celebrations. Locals and tourists take turns downing as many beers as possible as their friends shout Alpha Wars anthems.`
```

Shanties are used in a few places for the descriptions, but those are work songs, typically sung on ships to keep time doing a task like rowing. I don't think it really works to describe folk songs that people sing to remember an event. Depending on what you're going for, you could use 

- Ballads as recounting a story, often emotionally and a bit more somber.
- Laments or hymns for an even more solemn and perhaps more religious feel.
- Anthems as a more boisterous and uplifting celebratory song, which I think makes the most sense for the drunk locals/tourists here.

File: data/map planets.txt [hunk 9]
```diff
@@ -2381,6 +2422,10 @@ planet Ingot
...
 	spaceport `In this spaceport, ships dock inside a hangar whose ceiling protects them from meteorite impacts. The hangar is not pressurized, so there are also landing tubes that connect to each of the visiting ships. Many of the people inside are carrying emergency oxygen masks in case a sudden impact ruptures the building.`
 	spaceport `	In addition to being the spaceport, this complex is also the repair and control center for the mining robots. There are a handful of buildings elsewhere on Ingot, but not many. You cannot imagine spending so much time with so little space or change of scenery.`
+	spaceport `	The decorations for Remembrance Day on this planet amount to a batch of flags placed near trading hub. The spirit of the locals, however, is undeniable; you can feel the sound of shanties echo through the port.`
```
Comment (reticent-rem) [inline data/map planets.txt:2425]:
```suggestion
	spaceport `	The decorations for Remembrance Day on this planet amount to a batch of flags placed near trading hub. The spirit of the locals, however, is undeniable; you can feel the sound of anthems echo through the port.`
```
Alternatively chants or folk songs

File: data/map planets.txt [hunk 10]
```diff
@@ -3426,6 +3499,10 @@ planet "New India"
 	description `	The farms grow both food crops and cotton, and very recently they have begun manufacturing their own textiles instead of just shipping the raw cotton off-world for processing.`
 	spaceport `New India's spaceport is a ring of low wooden buildings around a wide landing area paved with concrete slabs. Around the perimeter carts of trade goods are driven, some being pulled by tractors and others, by horses or oxen. You even think you see a few carts being pulled by water buffalo.`
 	spaceport `	Most of the buildings are for storage, but from somewhere nearby you can smell fresh-baked bread.`
+	spaceport `	Despite the relative recency of settlement, Remembrance Day celebrations still appear to be occurring, with Alpha War shanties audible from one of the spaceport buildings.`
```
Comment (reticent-rem) [inline data/map planets.txt:3502]:
```suggestion
	spaceport `	Despite the relative recency of settlement, Remembrance Day celebrations still appear to be occurring, with Alpha War ballads audible from one of the spaceport buildings.`
```

File: data/map planets.txt [hunk 11]
```diff
@@ -4008,6 +4117,10 @@ planet Rand
...
 	description `	Aside from the managers of the mining companies, nearly all the people here are migrant workers from elsewhere in the Dirt Belt, who have come to spend a season working for the relatively high wages that uranium mining offers, either to send money off-world or to save it up in order to build a better life for themselves.`
 	spaceport `The landing pads here are poured concrete, raised up slightly above the level of the desert to keep them from being buried in sand. A few workers are constantly driving brush sweepers across the unoccupied pads. The air is filled with sand particles and dust. Aside from the storage sheds, there are only two buildings, each with a large sign out front: "Labor" and "Trade."`
+	spaceport `	Remembrance Day celebrations on Rand are sparse. It appears that most people have opted to return to their families on other planets to celebrate the day, though in one corner of the spaceport, you can hear a group of friends singing Alpha Wars shanties.`
```
Comment (reticent-rem) [inline data/map planets.txt:4120]:
```suggestion
	spaceport `	Remembrance Day celebrations on Rand are sparse. It appears that most people have opted to return to their families on other planets to celebrate the day, though in one corner of the spaceport, you can hear a group of friends singing Alpha Wars folk songs.`
```
Or ballads or anthems. Or hymns or laments if they're sad. Maybe 
```suggestion
	spaceport `	Remembrance Day celebrations on Rand are sparse. It appears that most people have opted to return to their families on other planets to celebrate the day, though in one corner of the spaceport, you can hear a small group singing Alpha Wars hymns, their voices carrying a mournful echo as they lament those lost.`
```

File: data/map planets.txt [hunk 12]
```diff
@@ -2134,6 +2167,10 @@ planet Heartland
...
 	description `	Tractors from Heartland are trusted to be well-built and sturdy, and easy to repair; some are still in use that were built over a century ago.`
 	spaceport `The spaceport is a cluster of low buildings. At the center of the port is a special, taller building with glass walls on all sides. It houses the tractor showroom. The farmers visiting the showroom look about as excited as you do when touring a large and well-appointed shipyard. There are tractors with rubber wheels, spiked wheels, and tracks; with plows, combines, balers, and chaser bins; tractors barely big enough to sit on and tractors the size of a small freighter. It's somewhat overwhelming.`
+	spaceport `	One section of the tractor showcase appears to currently be dedicated to antique tractors back from the Alpha Wars. On one landing pad, you spot some dock workers unloading a gigantic harvester out from a Bulk Freighter's hold.`
```
Comment (reticent-rem) [inline data/map planets.txt:2170]: In the repulsor & hovercraft focused present, the wheels help emphasize it's 500+ years (assuming its from the First Alpha War).
```suggestion
	spaceport `	One section of the tractor showcase appears to currently be dedicated to antique tractors back from the Alpha Wars. On one landing pad, you spot some dock workers unloading a gigantic wheeled harvester out from a Bulk Freighter's hold.`
```

File: data/map planets.txt [hunk 13]
```diff
@@ -5257,6 +5378,10 @@ planet Valhalla
 	description `	The shipyards of Lionheart Industries are located here. Centuries ago, warships built here turned the tide of the Alpha War, and today the ships designed and built in the Deep continue to be some of the most technologically advanced ships in human space - and the most expensive.`
 	spaceport `Surrounding the spaceport is a metropolis of skyscrapers and factories that spreads all the way to the horizon in every direction.`
 	spaceport `	The spaceport itself is an imposing building, composed of three towers rising over a hundred stories into the air and connected at various levels by bridges. The docking bays are on the higher levels, with warehouse space lower down and loading platforms for trucks at ground level. Massive cargo elevators run up and down in the center of each building, and the hallways are all wide enough to allow small robotic carts and forklifts in addition to foot traffic.`
+	spaceport `	Despite the Deep's importance to defeating the Alphas, Remembrance Day is seldom observed in the region. The only thing you see out of the ordinary is a milk dish on the counter of an information booth.`
```
Comment (reticent-rem) [inline data/map planets.txt:5381]: What's a milk dish?

**Review by reticent-rem — COMMENTED** [review]
Thanks for putting these together, I really like how it makes the world a bit more alive through time.

File: data/map planets.txt (hunk 13 above)
Comment (bene-dictator) [reply data/map planets.txt:5381]: It's a tradition of the people of the Deep to put out a dish of milk for the "elves". Like how kids put out cookies and milk for Santa.

File: data/map planets.txt (hunk 5 above)
Comment (bene-dictator) [reply data/map planets.txt:3368]: Just append another condition to take into account the start.

File: data/map planets.txt (hunk 9 above)
Comment (Anarchist2) [reply data/map planets.txt:2425]:
```suggestion
	spaceport `	The decorations for Remembrance Day on this planet amount to a batch of flags placed near trading hub. The spirit of the locals, however, is undeniable; you can feel folk songs echo through the metal corridors.`
```

File: data/map planets.txt (hunk 6 above)
Comment (Anarchist2) [reply data/map planets.txt:3450]: Without the change, the text ends up going too low. There's supposed to be at least a 10-pixel margin at the top and bottom of the box; having the extra text reduces the margin to 7 pixels at the bottom.

image

File: data/map planets.txt [hunk 14]
```diff
@@ -149,6 +154,10 @@ planet Allhome
 	landscape land/dmottl0
 	description `This Earth-like planet has been settled both by human beings and by the Hai, the species that controls this region of space. The human settlements are relatively small, mostly farming communities founded by people who came here to escape the chaos and uncertainty of human space. More substantial urban settlements exist, particularly near the spaceport, but from the air they appear to exist within strict boundaries.`
 	spaceport `In the spaceport, a handful of human merchants and colonists wander about amid throngs of Hai - vaguely squirrel-like aliens who are somewhat shorter than the average human being, but make up for it with an abundance of energy. You feel a little bit like an adult wading through a sea of hyperactive children as you try to locate the commodity exchange and the other services offered here.`
+	spaceport `	Despite the distance from Republic space, some humans have set up decorations in celebration of Remembrance Day. A handful of the more curious Hai approach the humans, asking them what the fuss is all about.`
```
Comment (TheGiraffe3) [inline data/map planets.txt:157]: Why just Republic space?

File: data/map planets.txt [hunk 15]
```diff
@@ -3426,6 +3499,10 @@ planet "New India"
 	description `	The farms grow both food crops and cotton, and very recently they have begun manufacturing their own textiles instead of just shipping the raw cotton off-world for processing.`
 	spaceport `New India's spaceport is a ring of low wooden buildings around a wide landing area paved with concrete slabs. Around the perimeter carts of trade goods are driven, some being pulled by tractors and others, by horses or oxen. You even think you see a few carts being pulled by water buffalo.`
 	spaceport `	Most of the buildings are for storage, but from somewhere nearby you can smell fresh-baked bread.`
+	spaceport `	Despite the relative recency of settlement, Remembrance Day celebrations still appear to be occurring, with Alpha War ballads audible from one of the spaceport buildings.`
```
Comment (TheGiraffe3) [inline data/map planets.txt:3502]:
```suggestion
	spaceport `	Despite the relative recency of settlement, Remembrance Day celebrations still appear to be occurring, with Alpha Wars ballads audible from one of the spaceport buildings.`
```

File: data/map planets.txt (hunk 14 above)
Comment (Anarchist2) [reply data/map planets.txt:157]: Because it's the closest region to Hai space. It's the difference between saying something is a certain distance from the mainland versus saying something is a certain distance from the shore, which is to say that they're basically the same.

**Review by TheGiraffe3 — APPROVED** [review]

File: data/map planets.txt [hunk 16]
```diff
@@ -2381,6 +2422,10 @@ planet Ingot
...
 	spaceport `In this spaceport, ships dock inside a hangar whose ceiling protects them from meteorite impacts. The hangar is not pressurized, so there are also landing tubes that connect to each of the visiting ships. Many of the people inside are carrying emergency oxygen masks in case a sudden impact ruptures the building.`
 	spaceport `	In addition to being the spaceport, this complex is also the repair and control center for the mining robots. There are a handful of buildings elsewhere on Ingot, but not many. You cannot imagine spending so much time with so little space or change of scenery.`
+	spaceport `	The decorations for Remembrance Day on this planet amount to a batch of flags placed near trading hub. The spirit of the locals, however, is undeniable; you can feel folk songs echo through the metal corridors.`
```
Comment (roadrunner56) [inline data/map planets.txt:2425]:
```suggestion
	spaceport `	The decorations for Remembrance Day on this planet amount to a batch of flags placed near a trading hub. The spirit of the locals, however, is undeniable; you can feel folk songs echo through the metal corridors.`
```

File: data/map planets.txt [hunk 17]
```diff
@@ -773,6 +794,14 @@ planet Clark
...
 	spaceport `The spaceport is at the heart of one of Clark's largest cities, surrounded by smokestacks and factories. There are a few hangars owned by different corporations where their cargo ships can load and unload, but you are relegated to a set of concrete landing pads farther from the main buildings. Large trucks rumble by carrying carts loaded down with cargo crates.`
 	spaceport `	In the central building, dozens of workers fill a large cafeteria with several stations serving different types of food. The mood here feels boisterous and energetic, very different from most of the Dirt Belt worlds you have visited.`
+		to display
+			or
+				"days since year start" <= 35
+				"days since year start" >= 55
+	spaceport `	In the central building, dozens of workers fill a large cafeteria with several stations serving different types of food. The mood here feels boisterous and energetic, bolstered further by Remembrance Day celebrations. Locals and tourists take turns downing as many beers as possible as their friends shout Alpha Wars folk songs.`
```
Comment (roadrunner56) [inline data/map planets.txt:801]:
```suggestion
	spaceport `	In the central building, dozens of workers fill a large cafeteria with several stations serving different types of food. The mood here feels boisterous and energetic, bolstered further by the local Remembrance Day celebrations. Locals and tourists take turns downing as many beers as possible as their friends shout Alpha Wars folk songs.`
```

File: data/map planets.txt (hunk 14 above)
Comment (mOctave) [inline data/map planets.txt:157]: Hai approaching the humans implies that a mission is about to be offered, and seems like an event instead of a description. Maybe:
```suggestion
	spaceport `	Despite the distance from Republic space, some humans have set up decorations in celebration of Remembrance Day. A handful of the more curious Hai mingle with the humans, asking them what the fuss is all about.`
```

File: data/map planets.txt [hunk 18]
```diff
@@ -223,6 +232,10 @@ planet Arabia
 	description `	Aside from hunting, most of the industry takes the form of harvesting and refining petrochemicals to make plastics, polymers, and oils. In some parts of Arabia, the pumpjacks outnumber the people a hundred to one.`
 	spaceport `The spaceport consists mostly of a wide, open field paved in asphalt, the cheapest material available here. The pavement shimmers with heat, and is pockmarked with divots where ships' landing gear has sunk several centimeters in to the surface.`
 	spaceport `	At the center of the pavement is a small cluster of refrigerated sheds and stacks of crates carrying various goods for trade; a few armed guards patrol around them to prevent theft. A food truck is parked nearby, serving burgers and tacos to the port workers and to the crews of the few starships that are parked here at the moment.`
+	spaceport `	Despite the climate, a small procession, consisting mainly of security guards and port workers, is gathered in the spaceport to celebrate Remembrance Day.`
```
Comment (mOctave) [inline data/map planets.txt:235]: This seems a bit comma-heavy. Maybe even:
```suggestion
	spaceport `	Despite the climate, a small procession of security guards and port workers is gathered in the spaceport to celebrate Remembrance Day.`
```

File: data/map planets.txt [hunk 19]
```diff
@@ -382,6 +395,10 @@ planet "Big Sky"
 	description `	The Big Sky government recently financed a major project to genetically engineer a goat species so ravenous that it can control the growth of local vegetation. These "ubergoats" pose so great a threat to the typical biosphere that they are export-controlled as biological weapons.`
 	spaceport `The landing pads here are being gradually consumed by the local plant life. In most places grass and ivy has crept out to cover at least a meter of the pad on all sides, and every small crack or crevice has been colonized by moss and by the roots of larger plants.`
 	spaceport `	In places the forest has been cleared to make roads leading to the nearby farming villages, but in most directions all you see is green. It is a beautiful sight, and oddly peaceful, but you know what a nuisance these plants are to the locals.`
+	spaceport `	A large gathering of locals is congregated around a memorial in the spaceport to commemorate Remembrance Day. The plaque and the surrounding path have been scoured clean of the native flora.`
```
Comment (mOctave) [inline data/map planets.txt:398]: This is a little unclear about whether the gathering is commemorating Remembrance Day, or the memorial; it reads like the first (probably) but is probably supposed to be both?

Also, saying a gathering is congregated is repetitive, which makes me feel like I'm missing something the first time I read it.

```suggestion
	spaceport `	A large number of locals have congregated around a memorial in the spaceport in order to commemorate Remembrance Day. The plaque and the surrounding path have been scoured clean of the native flora.`
```

File: data/map planets.txt [hunk 20]
```diff
@@ -2134,6 +2167,10 @@ planet Heartland
...
 	description `	Tractors from Heartland are trusted to be well-built and sturdy, and easy to repair; some are still in use that were built over a century ago.`
 	spaceport `The spaceport is a cluster of low buildings. At the center of the port is a special, taller building with glass walls on all sides. It houses the tractor showroom. The farmers visiting the showroom look about as excited as you do when touring a large and well-appointed shipyard. There are tractors with rubber wheels, spiked wheels, and tracks; with plows, combines, balers, and chaser bins; tractors barely big enough to sit on and tractors the size of a small freighter. It's somewhat overwhelming.`
+	spaceport `	One section of the tractor showcase appears to currently be dedicated to antique tractors back from the Alpha Wars. On one landing pad, you spot some dock workers unloading a gigantic wheeled harvester out from a Bulk Freighter's hold.`
```
Comment (mOctave) [inline data/map planets.txt:2170]:
```suggestion
	spaceport `	One section of the tractor showcase appears to currently be dedicated to antique tractors back from the Alpha Wars. On one landing pad, you spot some dock workers unloading a gigantic wheeled harvester out of a Bulk Freighter's hold.`
```

File: data/map planets.txt [hunk 21]
```diff
@@ -3426,6 +3499,10 @@ planet "New India"
 	description `	The farms grow both food crops and cotton, and very recently they have begun manufacturing their own textiles instead of just shipping the raw cotton off-world for processing.`
 	spaceport `New India's spaceport is a ring of low wooden buildings around a wide landing area paved with concrete slabs. Around the perimeter carts of trade goods are driven, some being pulled by tractors and others, by horses or oxen. You even think you see a few carts being pulled by water buffalo.`
 	spaceport `	Most of the buildings are for storage, but from somewhere nearby you can smell fresh-baked bread.`
+	spaceport `	Despite the relative recency of settlement, Remembrance Day celebrations still appear to be occurring, with Alpha Wars ballads audible from one of the spaceport buildings.`
```
Comment (mOctave) [inline data/map planets.txt:3502]: Despite/still is repetitive, which vaguely implies that the celebrations should have been finished by now.

File: data/map planets.txt [hunk 22]
```diff
@@ -3450,6 +3531,10 @@ planet "New Portland"
...
 	description `	Some of the farms near the spaceport are planted with different crops forming abstract patterns visible from the air. Such a whimsical extravagance seems particularly odd when you consider that the owners will never be able to afford to see their fields from that angle. But perhaps that is precisely the point.`
 	spaceport `The largest town on New Portland has set aside a landing area for ships, but it is barely what could be called a spaceport. Aside from the refueling station and a few sheds, there are no other buildings, just an open field of packed gravel, with grass and weeds peeking up in places where it has avoided the blast from landing spacecraft. A few farmers have parked wagons here and are trading food and other goods with the visiting ships.`
+	spaceport `	It appears that the spaceport has not been embellished for Remembrance Day. You assume that the main celebrations are taking place in the planet's individual towns.`
```
Comment (mOctave) [inline data/map planets.txt:3534]: This is borderline telling me what I think. It's probably okay, but you could also do something along the lines of:
```suggestion
	spaceport `	It appears that the spaceport has not been embellished for Remembrance Day. The main celebrations are likely taking place in the planet's individual towns.`
```

File: data/map planets.txt [hunk 23]
```diff
@@ -3539,6 +3636,10 @@ planet "New Washington"
 	description `	The only villages are in the twilight region in between, and there the settlers make a living by mining for metal and for oil, a relic of a warmer period when more life flourished here and the planet spun more quickly on its axis.`
 	spaceport `In this narrow ring between the baking sun of New Washington's day side, and the endless night beyond, a few native plants and animals still flourish: weirdly colored trees and grasses, and a few insect-like creatures flying among them.`
 	spaceport `	Rats, accidentally introduced by the settlers, also flourish here. A small, nearly deserted cafeteria is selling stew and mashed potatoes. You are almost certain the stew is made with rat meat.`
+	spaceport `	The cafeteria is littered with Remembrance Day memorabilia: flags of corporations from a long gone era; recruitment posters for a war that has ended; banners for a victory still ongoing. Judging by how haphazardly everything is strewn about, you assume that whoever put the decorations out must be rather lazy.`
```
Comment (mOctave) [inline data/map planets.txt:3639]: Semicolons would require a verb in each clause.

```suggestion
	spaceport `	The cafeteria is littered with Remembrance Day memorabilia: flags of corporations from a long gone era, recruitment posters for a war that has ended, banners for a victory still ongoing. Judging by how haphazardly everything is strewn about, you assume that whoever put the decorations out must be rather lazy.`
```

**Review by mOctave — COMMENTED** [review]
Love these!

File: data/map planets.txt (hunk 21 above)
Comment (Anarchist2) [reply data/map planets.txt:3502]:
```suggestion
	spaceport `	Despite the relative recency of settlement, Remembrance Day celebrations appear to be occurring, with Alpha Wars ballads audible from one of the spaceport buildings.`
```
